libs/langchain-cohere/src/chat_models.ts

```diff
@@ -145,6 +146,15 @@ export class ChatCohere<
     this.streaming = fields?.streaming ?? this.streaming;
   }
 
+  protected getLsParams(_: this["ParsedCallOptions"]): LangSmithParams {
+    return {
+      ls_provider: "cohere",
+      ls_model_name: this.model,
+      ls_model_type: "chat",
+      ls_temperature: this.temperature ?? undefined,
```

Comment:
this seems to be missing a bunch of params like max tokens, stop, etc see https://docs.cohere.com/reference/chat